bootloader/androidboot.go

```diff
@@ -54,16 +54,17 @@ func (a *androidboot) dir() string {
 
 func (a *androidboot) InstallBootConfig(gadgetDir string, opts *Options) error {
 	gadgetFile := filepath.Join(gadgetDir, a.Name()+".conf")
-	systemFile := a.ConfigFile()
+	systemFile, _ := a.ConfigFile()
```

Comment:
Even if we never return an error from newAndroidBoot right now I think we should still assign the error here (and in other bootloaders) and error if we get one (even if it does not matter today). Rational is a) maybe at some point ConfigFile will return an error at some point and then we are in for a surprise :) b) it just reads more naturally, i.e. no wonder "why do we ignore errors here".

Reply:
sure I can return error from all these such places